server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -542,23 +542,20 @@ protected void initializeExecutors()
     if (persistExecutor == null) {
       // use a blocking single threaded executor to throttle the firehose when write to disk is slow
       persistExecutor = Execs.newBlockingSingleThreaded(
-          "plumber_persist_%d", maxPendingPersists
+          "plumber_persist_%d", maxPendingPersists, Thread.MIN_PRIORITY
```

Comment:
We should make this configurable and leave it null by default. Let's have the user decide the priority at which to run the persist threads.